src/domain/tokens/__tests__/token.builder.ts

```diff
@@ -5,7 +5,7 @@ import { Token, TokenType } from '@/domain/tokens/entities/token.entity';
 export function tokenBuilder(): IBuilder<Token> {
   return new Builder<Token>()
     .with('address', faker.finance.ethereumAddress())
-    .with('decimals', faker.number.int())
+    .with('decimals', faker.number.int({ min: 0, max: 32 }))
```

Comment:
Nit: let's add a comment regarding the reasoning for this.

Reply:
Done in [5c4f7df](https://github.com/safe-global/safe-client-gateway/pull/1297/commits/5c4f7df798847b3130988e9f110ae806d89d8aeb), thanks!